tests/figures/chart/chart_plugin.test.ts

```diff
@@ -2223,6 +2223,77 @@ describe("Chart design configuration", () => {
     }
   );
 
+  test.each(["line", "bar", "combo"] as const)(
+    "%s chart does not display vertical grid lines",
+    (chartType) => {
+      setCellContent(model, "A1", "1");
+      setCellContent(model, "A2", "2");
+      setCellContent(model, "A3", "3");
+      setCellContent(model, "A4", "4");
+
+      createChart(
+        model,
+        {
+          dataSets: [
+            { dataRange: "A1:A2", yAxisId: "y" },
+            { dataRange: "A3:A4", yAxisId: "y1" },
+          ],
+          type: chartType,
+        },
+        "43"
+      );
+
+      const config = getChartConfiguration(model, "43");
+      expect(config.options?.scales?.x?.grid?.display).toBe(false);
```

Comment:
nitpick: all those tests could probably be simplified a bit, I don't think you need to set 4 cell content and to have 2 datasets in each chart